### ISSUE TITLE
ClusteringOrder doesn`t work in 1.8.0

### ISSUE THREAD

**Issue by coffius** [websudos/phantom#243]
Hi.

I think there is a problem with clustering order and schema generation.

I have the table that looks like:
```
class VisitorAffiliateLinkHitCasRecord extends CassandraTable[VisitorAffiliateLinkHitCasRecord, VisitorAffiliateLinkHit]{
  override lazy val tableName = "raw_visitor_affiliate_hits"

  //scalastyle:off object.name
  object visitor extends UUIDColumn(this) with PartitionKey[UUID]
  object ware extends LongColumn(this) with PrimaryKey[Long] with ClusteringOrder[Long] with Descending
  object partner extends LongColumn(this) with PrimaryKey[Long] with ClusteringOrder[Long] with Descending
  object date_time extends DateTimeColumn(this) with ClusteringOrder[DateTime] with Descending
  object link extends OptionalLongColumn(this)
  //scalastyle:on object.name
```

But when I try to generate schema
```
VisitorAffiliateLinkHitCasRecord.create.ifNotExists().future()
```

I have a wrong query in console:
```
CREATE TABLE IF NOT EXISTS test_keyspace.raw_visitor_affiliate_hits (visitor uuid, ware bigint, partner bigint, date_time timestamp, link bigint, PRIMARY KEY (visitor, ware, partner, date_time));
```
and table also has an invalid sort order

**Comment by alexflav23** [comment]
Hi @coffius,

Thank you for reporting this, it looks pretty bad. A fix will be pushed out today itself as this is a very important piece of functionality.

Regards.

**Comment by rwinzhang** [comment]
I found that the __"WITH CLUSTERING"__ clause is missing on the query and still happening on version __1.8.12__

**Comment by alexflav23** [comment]
@rwinzhang @coffius This has been fixed in version 1.9.3 and above, which is already on Bintray. Feel free to upgrade and everything should be working again.

**Comment by alexflav23** [comment]
Hi @rwinzhang @coffius You also have the full fix available in `1.9.5`. I will now close this ticket as latest releases will sort the problem out quickly.
